contexts/DonationContext/tests/Unit/UseCases/AddDonation/AddDonationUseCaseTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Unit\DonationContext\UseCases\AddDonation;
+
+use ReflectionMethod;
+use PHPUnit\Framework\TestCase;
+use WMDE\Fundraising\Frontend\DonationContext\UseCases\AddDonation\AddDonationUseCase;
+
+/**
+ * @covers \WMDE\Fundraising\Frontend\DonationContext\UseCases\AddDonation\AddDonationUseCase
+ */
+class AddDonationUseCaseTest extends TestCase {
+
+	public function testGetInitialDonationStatus(): void {
+		$useCase = $this->createMock( AddDonationUseCase::class );
+		$method = new ReflectionMethod( $useCase, 'getInitialDonationStatus' );
+		$method->setAccessible( true );
```

Comment:
It is not an approach I am happy with. If I rename a private method or somehow change the internal structure of a class, I don't want tests to break.

In case where dealing with the whole class is too cumbersome I prefer refactoring out the logic into a new class that can then be tested easier. And if the code is too trivial for that, then having a public static method (for functions that do not use state or have side effects) seems like a better trade off than using reflection to me.